singleval-novote/injective-1/cli/injectived-v1.16.4.sh

```
@@ -3,6 +3,7 @@
 # ulimit -n 120000
 
 docker run -it --rm \
+    -e DEVNET_FORCE_PASS_GOV_PROPOSALS=${DEVNET_FORCE_PASS_GOV_PROPOSALS:-true} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)